server/src/main/java/org/elasticsearch/cluster/metadata/DataStreamFailureStoreDefinition.java

```diff
@@ -109,7 +109,7 @@ public class DataStreamFailureStoreDefinition {
                     .startObject("error")
                     .startObject("properties")
                     .startObject("message")
-                    .field("type", "wildcard")
+                    .field("type", "match_only_text")
```

Comment:
I don't think we have to worry about these kinds of concerns yet since development is still guarded by a feature flag. Eventually we'll need to determine our BWC guidelines for updating these, but it's not likely to be any more complicated than the current guidelines for built in templates.